pkg/cloudprovider/cloud.go

```diff
@@ -80,12 +83,27 @@ type Instances interface {
 	List(filter string) ([]string, error)
 	// GetNodeResources gets the resources for a particular node
 	GetNodeResources(name string) (*api.NodeResources, error)
-	// Configure the specified instance using the spec
-	Configure(name string, spec *api.NodeSpec) error
-	// Delete all the configuration related to the instance, including other cloud resources
-	Release(name string) error
 }
 
+// Route is a representation of an advanced routing rule.
+type Route struct {
+	Name            string
+	TargetInstance  string
```

Comment:
Comment on this-- is it the hostname, ip address, or name-as-given-by-cloud-provider?